projects/storefrontlib/cms-components/product/product-tabs/product-reviews/product-reviews.component.html

```diff
@@ -149,7 +147,7 @@ <h3>{{ 'productReview.overallRating' | cxTranslate }}</h3>
         <label>
           <span class="label-content"
             >{{ 'productReview.rating' | cxTranslate }}
-            <ng-template [ngTemplateOutlet]="requiredAsterisk"></ng-template>
+            <cx-form-required-asterisks></cx-form-required-asterisks>
```

Comment:
With new angular versions, we can use open-close tags now.
ie. `<cx-form-required-asterisks />`